### DISCUSSION TITLE
"Send" URL to access the raw text content?

### DISCUSSION THREAD

**Discussion by thielj** [dani-garcia/vaultwarden#4538] [Q&A] [answered]
Hi

I would like to use the send feature to create a one-time secret for use in a script, like this for example:

```bash
SECRET=${curl -sSf 'https://myvault.cloud/some/url')
curl -sSf "https://some.other.site/protected/resource?secret=$SECRET" -o ...
```

Is there such a thing, without the need to parse through HTML?

**Comment by stefan0xC** [comment, marked as answer]
A Send would have to be decrypted by the client ([which is done by the page](https://bitwarden.com/blog/bitwarden-send-how-it-works/)) so I don't think this would work. You could use the Bitwarden CLI to open a Send, cf. https://bitwarden.com/help/send-cli/#receive
But I also think this is what something like the [Secrets Manager](https://bitwarden.com/help/secrets-manager-overview/) would be better suited for which Vaultwarden does not implement cf. https://github.com/dani-garcia/vaultwarden/discussions/4544#discussioncomment-9347398.

**Reply by thielj** [reply]
Thanks, never used the CLI. `bw send receive` might actually do the trick.
